main/main.ts

```diff
@@ -140,6 +140,15 @@ app.on('open-url', (e: Event, u: string) => {
     e.preventDefault();
     shell.openExternal(u);
 });
+app.once('will-finish-launching', function() {
```

Comment:
Why is 'will-finish-launching' callback hooked? Simply `app.once('open-file', ...)` on toplevel does not work?